hotness/bz.py

```diff
@@ -122,6 +122,13 @@ def follow_up(self, text, bug):
         self.bugzilla._proxy.Bug.update(update)
         self.log.info("Followed up on bug: %s" % bug.weburl)
 
+    def attach_log(self, filename, description, bug):
```

Comment:
It's unfortunate the old function is called ``attach_patch`` since this is basically identical. It might be better to just have new ``attach_file`` function that takes a ``patch`` keyword arg. ``attach_patch`` can then call that with ``patch=True`` and we can mark ``attach_patch`` as deprecated. Does that make sense?